pkg/api/api.go

```diff
@@ -413,13 +413,19 @@ func (hs *HTTPServer) registerRoutes() {
 		apiRoute.Post("/frontend-metrics", bind(metrics.PostFrontendMetricsCommand{}), routing.Wrap(hs.PostFrontendMetrics))
 
 		if hs.Live.IsEnabled() {
-			apiRoute.Post("/live/publish", bind(dtos.LivePublishCmd{}), routing.Wrap(hs.Live.HandleHTTPPublish))
+			apiRoute.Group("/live", func(liveRoute routing.RouteRegister) {
+				// the channel path is in the name
+				liveRoute.Post("/publish/*", routing.Wrap(hs.Live.HandleHTTPPublish))
```

Comment:
I think it's much better to avoid variadic `*` path for POST request whenever possible. Some reasons:
* JSON body with params follows existing API approach, simpler to follow and document
* JSON body with structured command allows to not worry too much about proper channel name encoding